system/emergency_handler/config/emergency_handler.param.yaml

```diff
@@ -0,0 +1,6 @@
+# Default configuration for emergency handler
+---
+/emergency_handler:
```

Comment:
Let's use `/**` instead of specifying node name so that parameter still applies even if the node name is changed in a launch file.

Reply:
Ah yeah good catch. I hadn't thought of that.